src/stability_sdk/utils.py

```diff
@@ -55,6 +55,13 @@
     'zero': generation.BORDER_ZERO,
 }
 
+INTERP_MODES = {
```

Comment:
We should probably genericize the interpolation api a bit more.
* 'mix' is the same as 'lerp' just in image space rather than latent space
* 'rife' is a specific frame interpolation model, which to me means it should probably be specified by an engine_id, otherwise we'll need to add an API type every time there's a frame interpolation innovation
* we should be able to specify latent interpolation in a way that's (at least mostly) agnostic to architecture. If we had built this a year ago, we'd be carrying around vestigial `gan-lerp` and `gan-slerp` protocols (or worse, invoking them as misnamed diffusion interpolators!)

Reply:
good idea. not sure yet which will survive and stick around.